exchange/howto/run-seednode.adoc

```diff
@@ -4,108 +4,306 @@
+
+## Get a seed node up and running
+
+You can run your seed node in many ways. The guide you are currently reading applies to a (debian-based) unix system with systemd available. Furthermore, we assume that only one seed node is operated per host. Please be aware that the guide is only a guide and not a step by step copy and paste how to, as we ignore for example the need for the occational root access and file permissions, swap, general opsec, among other things. In case your host differs from our reference system, we are sure you can handle yourself.
+
+There are four pieces to the puzzle of successfully running a seed node:
+
+- a Bitcoin full node
+- the Bisq seednode itself and
+- system health reports
+
+and last but not least, the all-important firewall.
+
+=== Firewall
+
+Although every port that is to be used only by localhost clients should be hardened by configuration and source code already, we recommend whitelisting stuff that is actually needed. Here is an example on how to configure iptables accordingly (the config drops any incoming connections except connection from localhost, SSH and responses.):
+
+	iptables -P INPUT DROP && iptables -A INPUT -i lo -j ACCEPT && iptables -A INPUT -p tcp --dport 22 -j ACCEPT && iptables -A INPUT -m state --state ESTABLISHED,RELATED -j ACCEPT
+
+[NOTE]
+====
+Make sure you execute *all `iptables` commands at once*! Executing only the first one is going to kick you out of your ssh session and does not let you back in.
+====
+
+=== Bitcoin Node
+
+Start by setting up a user for the bitcoin core stuff.
+
+	useradd -r -m bitcoind
+	cd /home/bitcoind
+
+Download the Bitcoin Core binaries https://bitcoin.org/en/download[here] to this directory, unpack it and, for updating convenience, create a symlink to the necessary binaries (so you only have to change the symlink on update):
+
+	tar xzf bitcoin-0.18.0-x86_64-linux-gnu.tar.gz
+	ln -s bitcoin-0.18.0/bin/bitcoind bitcoind
+	ln -s bitcoin-0.18.0/bin/bitcoin-cli bitcoin-cli
+
+Create a systemd service file `bitcoind.service` in the systemd service path or your operating system (something like `/usr/lib/systemd/system/`) and adapt it to your needs. We recommend to create a user `bitcoind` for service hardening reasons (). In the end, it should look like
+
+----
+[Unit]
+Description=Bitcoind
+After=network.target
+
+[Service]
+ExecStart=bitcoind -daemon \
+                   -printtoconsole \
+                   -nodebuglogfile \
+                   -pid=/home/bitcoind/.bitcoin/bitcoind.pid
+ExecStop=/home/bitcoind/bitcoin-cli stop
+
+Type=forking
+PIDFile=/home/bitcoind/.bitcoin/bitcoind.pid
+Restart=on-failure
+
+User=bitcoind
+Group=bitcoind
+
+PrivateTmp=true
+ProtectSystem=full
+NoNewPrivileges=true
+PrivateDevices=true
+MemoryDenyWriteExecute=true
+
+[Install]
+WantedBy=multi-user.target
+----
+
+Create a file `/home/bitcoind/.bitcoin/bitcoin.conf` that contains
+
+----
+server=1
+txindex=1
+dbcache=1000
```

Comment:
I am not familiar nor sure if this is necessary as none of my nodes specify this.

Reply:
I took these as input from the discussion on slack (bisq-seednode around 2019-07-21 noon UTC). May we have your options @sqrrm @mrosseel?

Reply:
I aligned the config to my working development config and removed the performance tweaks as well. I am leaving the `txindex=1` as it has been reported as needed by @sqrrm via slack.